module-services/service-db/include/service-db/DBServiceMessage.hpp

```diff
@@ -17,6 +17,13 @@ class DBServiceMessageBackup : public DBMessage
     std::string backupPath;
 };
 
+class DBServiceMessageSyncPackage : public DBMessage
+{
+  public:
+    DBServiceMessageSyncPackage(MessageType messageType, std::string syncPackagePath);
+    std::string syncPackagePath;
```

Comment:
this field has to be public?

Reply:
Yes, its needed here: `/products/PurePhone/services/db/ServiceDB.cpp:211`
However, I'll check if I can write it differently during https://appnroll.atlassian.net/browse/CP-1427